### DISCUSSION TITLE
SSID

### DISCUSSION THREAD

**Discussion by MrFortunaBeach** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#802] [Ideas]
I´m not a coder but is there a way in the code to add a fixed SSID with the password layzspam0dule. All modules get a SSID with different numbers.
BR
Thomas

**Comment by metaljay** [comment]
I think this was in a recent update as previous versions didn't change the SSID.

**Reply by MrFortunaBeach** [reply]
I found how to change it in the code. Thanks
